### ISSUE TITLE
Power Screen for Induction Matrix

### ISSUE THREAD

**Issue by PigeonBoy** [mekanism/Mekanism-Feature-Requests#291]
### Describe the feature you'd like

it would possibly be a 2x3(1 width) wall of blocks that shows a percentage, and the actual rf number to show how much is in the matrix without actually accessing it using it to be connected via a network(it would be in a gui for the screen and the matrix). therefore u can see how much is in it which is a very useful feature i think

### Describe alternatives you've considered

just a tiny screen block that goes on the outside of the matrix in place of a wall which i dont think would benefit almost anyone

### Additional context

it  would be super helpful to be able to lock it up in an underground box, but still be able to see if you need to make it bigger, or even if its always empty

**Comment by AlexHgC** [comment]
Mekanism V10.1 will works with Computer Craft, so you will be able to do this when it come

**Comment by PigeonBoy** [comment]
i asked for this because im too dumb to use computercraft and it would be simpler for everyone

**Comment by Mazlem** [comment]
Agreed, this should be built-in to the Induction Matrix, like the EnderIO capacitor banks where you can set blocks to show a capacity bar or in/out information.  And/or having a part like an item frame that you can attach to the Induction Matrix / Energy Cube / Universal Cable which shows that information.
